README.md

```diff
@@ -93,17 +105,21 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
+calling a function initiates it runs the function when called. referencing only grabs the information of that function.
 
-```
 ### Question #9
 
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
-```js
-// write code here
-```
+
+var object = {
+
+student = [nyron , 20]
+
+}
+
+sayHello = object.student;
```

Comment:
here, we want to define a method inside the `object` that prints out the `students` name when called.